packages/simcore-sdk/tests/unit/test_nodeports_v2.py

```diff
@@ -124,9 +123,8 @@ def e_tag() -> str:
 async def mock_upload_file(mocker, e_tag):
     mock = mocker.patch(
         "simcore_sdk.node_ports.filemanager.upload_file",
-        return_value=Future(),
+        return_value=("0", e_tag),
```

Comment:
So it is possible to not return a future which needs to be awaited for a result any longer?

Reply:
mock now automatically handles coroutines